### PR TITLE
Milestone 0: Initialize Book store

### PR DESCRIPTION
**Pull request by mohisa302**
### Initialize Book store
In this milestone I:
- Added books and configuration pages and routers
- Added main page UI
- Added form

### CODE REVIEW

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalid Request* :red_circle:

Hi @mohisa302, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] has wrong workflow. You're required to use GitFlow not GitHub Flow. Kindly change that.

gitflow


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/bookStoreApp.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+const BookStoreApp = () => (
+  <div className={styles.pageContainer}>
+    <Routes>
+      <Route path="/" element={<Navbar />}>
+        <Route path="/Books" element={<Books />} />
+        <Route path="/" element={<Navigate to="/Books" />} />
+        <Route path="/Categories" element={<Categories />} />
+        <Route path="*" element={<NotMatch />} />
+      </Route>
+    </Routes>
```
Comment (Ismailco) [inline src/components/bookStoreApp.jsx]: - [x] This code should be present in the `App` component directly, kindly make sure to remove this component and use the `App` component to set up Routes

File: src/components/Book.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import styles from '../styles/Book.module.css';
+import progIcon from '../images/progress.jpg';
+
+const Book = () => {
```
Comment (Ismailco) [inline src/components/Book.jsx]: - [ ] The book data should pass to this component using `props` from the parent component, kindly implement this to use `props` instead

File: src/components/Categories.jsx [hunk 3]
```diff
@@ -0,0 +1,4 @@
+import styles from '../styles/Books.module.css';
+
+const Categories = () => <div className={styles.BOOKS}>Categories</div>;
```
Comment (Ismailco) [inline src/components/Categories.jsx]: - [ ] The `Categories` component should include a `button` with the text "Check status", kindly consider creating a button here so the user has a way to check the status (for now the button will not have any functionality)

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @mohisa302, 

Good job so far 👍🏼 
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights

✔️ No linter errors
✔️ Use the correct workflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Categories.jsx (hunk 3 above)
Comment (mohisa302) [reply src/components/Categories.jsx:3]: Thanks Ismailco I applied changes

Comment (mohisa302) [reply src/components/Categories.jsx]: Thanks, Ismailco I applied the changes

**Review by iambenkis — APPROVED** [review]
Hi @mohisa302 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@iambenkis** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**